### DISCUSSION TITLE
HELP: YOLOV5 Custom Dataset, How to deploy in Raspberry Pi 4B

### DISCUSSION THREAD

**Discussion by Ramilbadboy** [ultralytics/yolov5#13068] [Q&A]
I'm a beginner in raspberry pi and YOLO, I already trained my custom dataset and install the needed dependencies to the RPi. I already downloaded the best.pt (from colab), and pasted it to the yolov5 folder in RPi. But somehow when I ran the detect.py, the labels, datasets that were being used is from COCO. I also configure the detect.py but still nothing changed.
![20240521_11h58m47s_grim](https://github.com/ultralytics/yolov5/assets/171836593/451e780c-9f6e-4193-a69f-6efc41446663)

**Comment by kronee0516** [comment]
What do you mean? the result show the classes are from coco? the inference result image is coco?
